src/main/python/ttconv/tt.py

```diff
@@ -31,12 +31,45 @@
 from argparse import ArgumentParser
 import xml.etree.ElementTree as et
 from pathlib import Path
+from enum import Enum
+import typing
 import ttconv.imsc.reader as imsc_reader
 import ttconv.imsc.writer as imsc_writer
 import ttconv.scc.reader as scc_reader
 
 LOGGER = logging.getLogger(__name__)
 
+class FileTypes(Enum):
+  '''Enumerates the types of supported'''
+  TTML = "ttml"
+  SCC = "scc"
+
+  @staticmethod
+  def str_to_type(file_type_str: str):
+    return {
```

Comment:
Can be replaced by:

```
FileType("ttml")
```

Note that it can throw `ValueError`

Reply:
Done